### DISCUSSION TITLE
GitHub Mobile Language Settings Unavailable

### DISCUSSION THREAD

**Discussion by Cynthia7979** [community/community#30426] [Mobile] [answered]
A "Language" option exists on the GitHub mobile settings list. However, clicking on it sends the user to GutHub's page in the iOS Settings APP, which does not have a language option.

Steps to reproduce:
1. Press the settings icon in GitHub Mobile
2. Press Language
3. No language option available on the landing page

I'm currently using Chinese as my device language and would like to use GitHub in English. I don't want to change device-wide preference to achieve this, so that's why I'm trying to access the language settings.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, 版本 1.76.0 (75005810)
Device: iPad Pro 10.5-inch (iOS 14.8.1)
</details>

**Comment by mxie** [comment]
Hi @Cynthia7979 - I'm sorry to hear you're having issues changing your language. Unfortunately, I was not able to reproduce this on the iPad or iPhone. Tapping on the `Language` option did bring me to the settings screen and it has a section called `Preferred Language`. It looks something like this:
<img src="https://user-images.githubusercontent.com/602470/186190956-9c995054-53b0-4af7-a4d3-76844b9c3f80.jpeg" width="25%">

Does that option not appear for you?

**Reply by Cynthia7979** [reply]
Hi @mxie : Thanks for the reply. I do have this option on my iPhone (iOS 15.6.1) but not my iPad (iPadOS 14.8.1). Maybe it's a system version issue then?

**Reply by HenSquared** [reply]
Hi @Cynthia7979 , I think I figured this out. My screenshots are from an iPhone but should be the same on any device running iOS 13 and above. 

The language option only appears if you have more than one preferred language on your device. 

On your iPad, go into Settings > General > Language & Region > Other Languages 

This will display a list of languages. Choose the additional language you'd like to use. 

![image](https://user-images.githubusercontent.com/92610393/186539523-c851968d-7de3-47b8-9bbe-80c8b4788dab.png)

You can choose to have the new language as your phone's main language by choosing "Change to [Language]"

or you can just add the additional language without changing system language by choosing "Keep [Language"]

Your device may turn black for a second. 

Now you should be able to go to Settings > GitHub and see the "Preferred language" section. 

![image](https://user-images.githubusercontent.com/92610393/186539710-91fa9ca6-7d0e-440d-8d76-6a185b7eb636.png)

**Reply by Cynthia7979** [reply]
Hi @HenSquared - that works! Thank you so much for the help, much appreciated 😀 

By the way, would you mind putting this solution in a seperate post so that I could mark it as answer?

**Reply by alialhammad1992** [reply]
Is it possible to add Arabic, is there a way? iPhone 13 Pro Max

iOS 16.0

**Reply by Cynthia7979** [reply]
@alialhammad1992 Try to follow @HenSquared 's steps and add Arabic as one of your iPhone's languages. After that you can change the display language by clicking into the Language setting in GitHub Mobile.

**Comment by HenSquared** [comment, marked as answer]
Copying to mark as answer::


Hi, I think I figured this out. My screenshots are from an iPhone but should be the same on any device running iOS 13 and above. 

The language option only appears if you have more than one preferred language on your device. 

On your iPad, go into Settings > General > Language & Region > Other Languages 

This will display a list of languages. Choose the additional language you'd like to use. 

![image](https://user-images.githubusercontent.com/92610393/186539523-c851968d-7de3-47b8-9bbe-80c8b4788dab.png)

You can choose to have the new language as your phone's main language by choosing "Change to [Language]"

or you can just add the additional language without changing system language by choosing "Keep [Language"]

Your device may turn black for a second. 

Now you should be able to go to Settings > GitHub and see the "Preferred language" section. 

![image](https://user-images.githubusercontent.com/92610393/186539710-91fa9ca6-7d0e-440d-8d76-6a185b7eb636.png)
